### ISSUE TITLE
Cannot find module 'array-sort-by' from '...'

### ISSUE THREAD

**Issue by codeuniquely** [jherax/array-sort-by#6]
I have a number of React Jest / Enzyme tests which test a component that uses array-sort-by

i include the following (naming my default import as arraySort)
```
// make sorting by types and nested object easier
import arraySort from 'array-sort-by';
```

and when i have version 1.1.0 installed everything works as expected and the test all pass. However when I upgrade to version 1.2.0 then the test fail  to run with the error message above....

The code still compiles  but fails to run with a
 (0 , _arraySortBy.sortBy) is not a function

What do I need to import in order to get version 1.2.0 importing / running - as before ?

**Comment by jherax** [comment]
@codeuniquely I am checking this...
I will notify you when finished.

**Comment by jherax** [comment]
It's my fault, I renamed the `/dist` files, so in the `package.json` the name is wrong, I will be fixing this...

**Comment by codeuniquely** [comment]
Cheers .....

**Comment by jherax** [comment]
@codeuniquely Released fix in the version 1.2.1

```javascript
import sortBy from 'array-sort-by';

var arr = ['único', 'cosas', 'Árbol', 'fútbol', 'algo'];
sortBy(arr, item => 'desc:' + item);
/**
 * expected:
 * ["único", "fútbol", "cosas", "Árbol", "algo"]
 */
```
